Source/WebCore/css/calc/CSSCalcTree.h

```diff
@@ -24,6 +24,7 @@
 
 #pragma once
 
+#include "AnchorPositionEvaluator.h"
```

Comment:
It should be possible to avoid this include by forward declaring the enumeration value.

Reply:
Yeah this has been sticking out like a sore thumb. I didn't know enums can be forward declared, I'll do that.